app/resources/forms.js

```diff
@@ -158,6 +158,8 @@ function translate(source, target, text, continuation) {
       } else if (target.length > 2) {
         // Try falling back to "fr" if "fr-CA" didn't work.
         translate(source, target.slice(0, 2), text, continuation);
+      } else {
+        console_error("Call to Translate API has failed: ", response);
```

Comment:
With the fix in base.html.template, translate() should no longer be called if the translate_api_key is not set, with check here:
https://github.com/google/personfinder/blob/ichikawa-translate-no-key/app/resources/forms.js#L215

The check didn't work as expected before because string 'None' was set.